docs/notable/README.md

```diff
@@ -0,0 +1,15 @@
+# Thinks worth taking note of
+Just a dump of information that can be worth taking note of, as a new developer on the team it can be smart to look through some of these resources.
+
+## Security groups for pods
+
+There are some things worth considering about security groups for pods, the following is a small sample of the information you can find [here](https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html):
+
+- Security groups for pods can't be used with pods deployed to Fargate.
```

Comment:
:cry: 

Does that mean that pods on Fargate always get access to the DB, or never? (If we don't know, we should find out.)

And can mention whatever the answer to that here?

Reply:
Well, that depends, but for us it means never. I have added documentation